README.md

```diff
@@ -1,124 +1,115 @@
 
-- Build **Heapster** container
-```
-cd kubesnap/src/heapster
-docker build -t heapster-snap .
-docker tag heapster-snap <docker_registry>/heapster-snap:1
-docker push <docker_registry>/heapster-snap:1
-```
+  ![Alt text](docs/images/image_02.png "Google Cloud Console Menu") 
 
-- Build **Influxdb** container
-```
-cd heapster/influxdb
-docker build -t influxdb-snap .
-docker tag influxdb-snap <docker_registry>/influxdb-snap:1
-docker push <docker_registry>/influxdb-snap:1
-```
+- and then select _Compute Engine_ from the drop-down list
 
-- Build **Grafana** container
-```
-cd heapster/grafana
-docker build -t grafana-snap .
-docker tag grafana-snap <docker_registry>/grafana-snap:1
-docker push <docker_registry>/grafana-snap:1
-```
+  ![Alt text](docs/images/image_03.png "Selecting Compute Engine in Google Cloud Console Menu")
 
-- Build **snap** container
-```
-cd kubesnap/src/snap
-docker build -t snap .
-docker tag snap <docker_registry>/snap:1
-docker push <docker_registry>/snap:1
-```
+#### d) Create an VM instance  
+- create a new VM instance
+  ![Alt text](docs/images/image_04.png "Creating an VM instance")  
 
-- (Optional) Build **workload** container
-```
-cd kubesnap/src/workload
-docker build -t workload .
-docker tag workload <docker_registry>/workload:1
-docker push <docker_registry>/workload:1
-```
+- set an unique name of a new instance
+- choose a machine with 4 vCPUs (or more) and at least 15GB RAM
+- select Ubuntu 16.04 with standard persistent disk with at least 100GB
 
-- (Optional) Build **snap_stub** container
-```
-cd kubesnap/src/snap_stub
-docker build -t snap_stub .
-docker tag snap_stub 10.1.23.1:5000/snap_stub:1
-docker push 10.1.23.1:5000/snap_stub:1
-```
+  ![Alt text](docs/images/image_05.png "Setting up a new VM")  
 
-#### Running
+#### e) Start GCE Virtual Machine  
+- select your VM and choose _START_ option which is available on the top of screen
 
-- Deploy snap (or snap_stub)
-```
-cd kubesnap/deploy
-kubectl create -f snap/ OR kubectl create -f snap_stub/
-```
+  ![Alt text](docs/images/image_06.png "Staring an VM instance")  
+
+#### f) Open the VM terminal by click on SSH  
+ -  click on SSH to open the VM terminal (it will open as a new window)
+
+  ![Alt text](docs/images/image_07.png "Opening the terminal of Virtual Machine") 
+
+#### g) Authorize access to Google Cloud Platform  
+- manage credentials for the Google Cloud SD. To do that, run the following command:
+  ```
+  gcloud auth login
+  ```
+  Answer `Y` to the question (see below) and follow the instructions:
+  -	copy the link in your browser and 
+  -	authenticate with a service account which you use in Google Cloud Environment,
+  - copy the verification code from browser window and enter it
 
-- Deploy Heapster, InfluxDB and Grafana
+  ![Alt text](docs/images/image_08.png "Access authorization to Google Cloud Platform")
+
+- check if you are on credentialed accounts:  
```

Comment:
Accordance with the documentation: `gcloud auth list - lists credentialed accounts`